src/main/java/frc/robot/RobotContainer.java

```diff
@@ -58,10 +59,34 @@ public class RobotContainer {
     private double MaxSpeed = TunerConstants.kSpeedAt12Volts.in(MetersPerSecond); // kSpeedAt12Volts desired top speed
     private double MaxAngularRate = RotationsPerSecond.of(.825).in(RadiansPerSecond); // 3/4 of a rotation per second max angular velocity
 
+    // #region driver_tuning
+    // Flag to enable/disable driver tuning
+    private static final boolean ENABLE_DRIVER_TUNING = false;
+
+    private double translationDeadband = 0.0; // 0% default deadband to match current behavior
```

Comment:
As configured this should be a NOOP change (same deadbands for controller/drivetrain)